cmake/std/atdm/ATDMDevEnvSettings.cmake

```diff
@@ -202,15 +207,21 @@ ENDIF()
 # E) Set up other misc options
 #
 
-# Currently, EMPIRE configures of Trilinos have this enabled by default.  But
-# really we should elevate every subpackage that ATDM uses to Primary Tested.
-# That is the right solution.
-ATDM_SET_ENABLE(Trilinos_ENABLE_SECONDARY_TESTED_CODE ON)
+IF (ATDM_PT_PACKAGES)
+  ATDM_SET_ENABLE(Trilinos_ENABLE_SECONDARY_TESTED_CODE OFF)
+ELSE()
+  # Currently, EMPIRE configures of Trilinos have this enabled by default.  But
+  # really we should elevate every subpackage that ATDM uses to Primary Tested.
+  # That is the right solution.
+  ATDM_SET_ENABLE(Trilinos_ENABLE_SECONDARY_TESTED_CODE ON)
```

Comment:
do we know what's not PT that empire uses?

Reply:
@rppawlo asked:

> do we know what's not PT that empire uses?

That is what needs to be nailed down in #2597.   We already know of some for SPARC and Gemma.  For some reason, the EMPIRE configuration was explicitly setting Trilinos_ENABLE_SECONDARY_TESTED_CODE=ON.  

It would not be hard to figure that out what ST SE packages (if any) EMPIRE is using that are not PT SE packages.   Some quick configure experiments setting Trilinos_ENABLE_SECONDARY_TESTED_CODE to on an off would tell you that.

So many things to do ...